test/run

```diff
@@ -4,6 +4,8 @@
 
 set -eu
 
+export RUN_TESTS_OPTIONS=--track-naughties
```

Comment:
Why do we export that option here instead `--track-naughties` directly at Makefile. Do we expect more logic to be added to `test/run` which will make RUN_TESTS_OPTIONS options more dynamic/variable?

Reply:
Because the idea of that starter-kit commit was that running `make check` as a developer does *not* talk to github. For most people it doesn't work anyway, and it's somewhat unexpected even for us.